content/400-pulse/250-database-setup/500-supabase.mdx

```diff
@@ -8,17 +8,12 @@ toc: true
 
 You have to use a direct database connection from Supabase to use it with Prisma Pulse.
 
-## Using a direct connection in Supabase
+## Getting a direct connection string from Supabase for Pulse
 
-To use a direct database connection string in Supabase:
+1. Follow the Supabase docs and copy the direct [database connection string](https://supabase.com/docs/guides/database/connecting-to-postgres#direct-connections). Use this to enable Pulse for your project environment.
 
-1. Navigate to your Project settings (**Settings** > **Configurations** > **Database**).
-2. Disable the **Use connection pooling** option.
-   ![Supabase direct database connection](/img/pulse/supabase-direct-database-connection.png)
-3. Copy the direct database connection string.
+    :::info
+    Prisma Pulse is currently incompatible with Supabase's connection pooler because it intercepts communication between your database and Prisma Pulse, disrupting the [Streaming Replication Protocol](https://www.postgresql.org/docs/current/protocol-replication.html#PROTOCOL-REPLICATION). This protocol is used by Prisma Pulse to transmit replication commands to the database.
```

Comment:
```suggestion
    Prisma Pulse is currently incompatible with Supabase's connection pooler because Supabase intercepts communication between your database and Prisma Pulse, disrupting the [Streaming Replication Protocol](https://www.postgresql.org/docs/current/protocol-replication.html#PROTOCOL-REPLICATION). This protocol is used by Prisma Pulse to transmit replication commands to the database.
```